python/cuml/dask/ensemble/base.py

```diff
@@ -264,6 +287,17 @@ def _func_fit(model, input_data, convert_dtype):
     return model.fit(X, y, convert_dtype)
 
 
+def _func_predict(model, input_data, **kwargs):
+    X = concatenate(input_data)
+    with cuml.using_output_type("numpy"):
+        prediction = model.predict(X, **kwargs)
+        return np.expand_dims(prediction, axis=1)
+
+
+def _print_summary_func(model):

```

Comment:
Not sure why this was needed? I thought Philip already added a function to handle this?